_layouts/base.html

```diff
@@ -39,8 +38,34 @@
           'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
     })(window, document, 'script', 'dataLayer', 'GTM-TDV3PRK');</script>
   <!-- End Google Tag Manager -->
-</head>
 
+  <!-- link rel="stylesheet"-->
+  <link rel="stylesheet" href="/css/styles.css" />
```

Comment:
I would move style this above any GA scripts tags. We don't want to block the rendering of the page for GA tracking.

Inline JS should only be before CSS when it will alter the presentation of the page - i.e. adding a presentational class which would mean the page would need to repaint the CSS.

Reply:
so right after is ok?